draft-ietf-tls-rfc4492bis.xml

```diff
@@ -101,7 +101,8 @@
         generation of bulk encryption/MAC keys and initialization vectors is independent of the key exchange
         algorithm and not impacted by the introduction of ECC.</t>
       <t> <xref target="tbl2"/> summarizes the new key exchange algorithms.  All of these key exchange algorithms
-        provide forward secrecy.</t>
+        provide forward secrecy if and only if fresh ephemeral keys are generated and used, and also destroyed
```

Comment:
My point was just that "fresh" was confusing because often we use that to mean "one-use".